cmake/cuda.cmake

```diff
@@ -16,7 +16,7 @@ else()
   set(paddle_known_gpu_archs8 "30 35 50 52 60 61")
   set(paddle_known_gpu_archs9 "30 35 50 52 60 61 70")
   set(paddle_known_gpu_archs10 "30 35 50 52 60 61 70 75")
-  set(paddle_known_gpu_archs11 "35 50 52 60 61 70 75 80")
+  set(paddle_known_gpu_archs11 "52 60 61 70 75 80")
```

Comment:
if 50 is needed?

Reply:
for A100 50 is not supported， so we deperate it